setup/azurecmdfiles.wxi

```diff
@@ -799,6 +799,9 @@
                         <Component Id="cmp5835D87A5A94F44EAAAE61BCA33C1D80" Guid="*">
                             <File Id="filEC64A002E6CBA598F7148F816DB05F11" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Azure.Commands.DataLakeAnalytics.dll" />
                         </Component>
+                        <Component Id="cmp96C621A371FF4A410548AAB167869D94" Guid="*">
+                            <File Id="filC7912B6B895CF7DF00A7C7F821AE40BF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Azure.Commands.DataLakeAnalytics.dll-help.xml" />
```

Comment:
you should not have DataLake dependency changes. please run git clean -dfx and then run BuildInstaller.ps1 from tools folder again.